linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/utils.js

```diff
@@ -134,20 +134,26 @@ UpdateableChart.prototype.setMetric = function(metric) {
 UpdateableChart.prototype.setMetrics = function(metrics) {
   clearTimeout(this.timeout);
 
-  if (this.ts !== undefined) {
-    this.chart.removeTimeSeries(this.ts);
+  if (this.tsMap !== undefined) {
+    _.map(this.tsMap, function(ts){
+      this.chart.removeTimeSeries(ts);
+    });
   }
-  this.ts = new TimeSeries();
-  this.chart.addTimeSeries(
-    this.ts,
-    {
-      strokeStyle: 'rgb(83,176,196)',
-      fillStyle: 'rgba(83,176,196,0.3)',
-      lineWidth: 3
-    }
-  );
-
-  this._getMetrics(metrics);
+
+  this.tsMap = {};
+
+  _.each(metrics, function(metric) {
+    this.tsMap[metric.name] = new TimeSeries();
+    this.chart.addTimeSeries(
```

Comment:
the `router clients` and related sections auto-update when clients are added/removed. i think the chart should at least add clients as they are created, removal is less important.